manifests/cf-manifest/scripts/generate-cf-secrets.rb

```diff
@@ -30,6 +30,7 @@
   "loggregator_endpoint_shared_secret" => :simple,
   "consul_encrypt_keys" => :simple_in_array,
   "grafana_admin_password" => :simple,
+  "rds_broker_admin_password" => :simple,
```

Comment:
This will need to be added to the test fixtures as well (that's why the Travis build is currently failing).